cedar-policy-core/src/entities/json/err.rs

```diff
@@ -119,8 +119,8 @@ pub enum JsonDeserializationError {
         &.uid.entity_type(),
         match .suggested_types.as_slice() {
             [] => String::new(),
-            [ty] => format!("; did you mean {ty}?"),
-            tys => format!("; did you mean one of {:?}?", tys.iter().map(ToString::to_string).collect::<Vec<String>>())
+            [ty] => format!(". Did you mean {ty}?"),
```

Comment:
should suggested type be surrounded in backticks?

Reply:
Yep